mobile/components/UserEntry.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { FontAwesome, Ionicons } from "@expo/vector-icons";
+import { TouchableOpacity, StyleSheet, View, Text } from "react-native";
+import IconButton from "./IconButton";
+
+const UserEntry = (props: any) => {
+  return (
+    <TouchableOpacity onPress={undefined} style={styles.userEntryContainer}>
+      <View style={styles.userEntry}>
+        <FontAwesome name="user-circle" size={24} color="grey" />
+        <Text style={styles.userLogText}>User Name</Text>
+        {props.isVerification ? (
+          <View style={styles.userLogIcon}>
+            <IconButton
+              icon={<Ionicons name="checkmark-circle" size={30} color="grey" />}
+            ></IconButton>
+            <IconButton
+              icon={<Ionicons name="close-circle" size={30} color="grey" />}
+            ></IconButton>
+          </View>
+        ) : (
+          <Text style={styles.userLogText}>test@gmail.com</Text>
```

Comment:
make the email a prop you can pass in -- no email is specified default to "No Email Found"